README.md

```diff
@@ -11,15 +11,69 @@ npm install react-native-network-client
 ## Usage
 
 ```js
-import GenericClient, {getOrCreateAPIClient, getOrCreateWebSocketClient} from "react-native-network-client";
+import GenericClient, {
+  getOrCreateAPIClient,
+  getOrCreateWebSocketClient,
+} from "react-native-network-client";
 
 // ...
 
 const response = await GenericClient.get("https://community.mattermost.com");
-const {client: apiClient, created} = await getOrCreateAPIClient("https://community.mattermost.com");
-const {client: wsClient, created} = await getOrCreateWebSocketClient("wss://community.mattermost.com");
+const { client: apiClient, created } = await getOrCreateAPIClient(
+  "https://community.mattermost.com"
+);
+const { client: wsClient, created } = await getOrCreateWebSocketClient(
+  "wss://community.mattermost.com"
+);
 ```
 
+## Errors
+
+There are two cases where errors will be returned by the native code. The first is via rejected promises and the second is via events.
```

Comment:
Isn't the second one really using an error callback triggered by the JS side of the lib? I tought only the Lib was listening for the events, maybe I'm wrong?